s3_storage_client.go

```diff
@@ -38,11 +39,34 @@ type s3Location struct {
 	s3Path     string
 }
 
+var S3LoggingMode aws.ClientLogMode
+
 func (util *snowflakeS3Client) createClient(info *execResponseStageInfo, useAccelerateEndpoint bool) (cloudClient, error) {
 	stageCredentials := info.Creds
-	var resolver s3.EndpointResolver
+	s3Logger := logging.LoggerFunc(func(classification logging.Classification, format string, v ...interface{}) {
+		if len(v) > 1 {
+			if f, ok := v[0].(string); ok {
+				switch classification {
+				case logging.Debug:
+					logger.WithField("process", "S3").Debugf(f, v[1:]...)
```

Comment:
is "process" the best name for this field? Can we extract it to a const?

Reply:
I don't like it too, do you have a better name?

Reply:
Changed to `logger`.